js/web/script/build.ts

```diff
@@ -33,6 +33,8 @@ const FILTER = args.f || args.filter;
 const ROOT_FOLDER = path.join(__dirname, '..');
 const WASM_BINDING_FOLDER = path.join(ROOT_FOLDER, 'lib', 'wasm', 'binding');
 const WASM_BINDING_JS_PATH = path.join(WASM_BINDING_FOLDER, 'ort-wasm.js');
+const TRAINING_WASM_BINDING_SIMD_JS_PATH = path.join(WASM_BINDING_FOLDER, 'ort-training-wasm-simd.js');
+const TRAINING_WASM_BINDING_SIMD_MIN_JS_PATH = path.join(WASM_BINDING_FOLDER, 'ort-training-wasm-simd.min.js');
```

Comment:
what is this for? min build?

Reply:
yes, it can be used for min build. I was having issues earlier with Node running out of memory so I thought that reducing the JS artifact size could help.